### DISCUSSION TITLE
Sous-dossier "Api" en trop : Controller/Api/XXXXXController

### DISCUSSION THREAD

**Discussion by BaptisteDaSilva** [nilsteampassnet/TeamPass#3571] [General]
Bonjour,

Il faut supprimer le dossier "Api" (Controller/**Api**/XXXXXController) pour que les modifications suivantes fonctionnent :

![image](https://user-images.githubusercontent.com/35465700/222968147-4abe7928-991b-48ba-8df1-98ff9d18de69.png)

![image](https://user-images.githubusercontent.com/35465700/222968302-f01fca29-72e8-4b62-b49f-66eead297da0.png)

![image](https://user-images.githubusercontent.com/35465700/222968316-6877ef12-c56f-4860-a7dd-4b337d6cde37.png)

Cordialement,

Baptiste

**Comment by nilsteampassnet** [comment]
BOnjour

Ça doit être fait lors d'un des derniers commits.

MErci
